tests/ctst/features/pra.feature

```diff
@@ -22,6 +22,10 @@ Feature: PRA operations
 
     # Check that objects are transitioned in the DR site
     Given access keys for the replicated account
+
+    When the DATA_ACCESSOR user tries to perform PutObject on "DR" site
+    Then it "should not" pass Vault authentication
```

Comment:
Are we sure this won't be flaky? As we just deployed DR at this stage, the "access keys for the replicated account" may complete quickly with no guarantee that the data accessor was already replicated
While having this check after the checks for the transitioned objects, we benefit from the waiting times

Reply:
If drsink and drsource are running everything is replicated

Reply:
we do the check after the drsink and drsource are in running phase, which means everything is replicated, if something is not then its a bug and we want the test to fail

Reply:
ye sok replicated through the initial dump, but not sure this is enough if the AK was created after (it may need time to replicated)
But didn't you get an issue with this test at first, and then it worked?

Reply:
AK are created on the dr site itself, i am not sure i understand what you want to be replicated exactly ?
Never had such an issue AFAIR

Reply:
Sorry I meant the role, not AK; the role will need to be replicated at the time this step runs, and having the step "running" is not enough iff the role was created after the dump, but that shouldn't be the case